### ISSUE TITLE
[Bug] [Beta] Janky behavior when hovering over Play/More buttons on modpack card

### ISSUE THREAD

**Issue by aaronhowser1** [FTBTeam/FTB-App#189]
**Operating System:**
OS: W10

**Installer Version:**
Installer: FTBA_windows-x64_202007042254-9878310b16-develop

![image](https://user-images.githubusercontent.com/15698722/86527317-790a9d80-be63-11ea-9843-3f41baf0c918.png)
Can't figure out how to copy these and I'd rather not manually transcribe

**Log Files:**
https://pste.ch/abanipitit

**Describe the bug**
![ezgif-6-5feff59574b6](https://user-images.githubusercontent.com/15698722/86527331-99d2f300-be63-11ea-9fd8-d3f6465a533d.gif)

**To Reproduce**
1. Move your mouse around the play/more buttons

**Comment by ThePaul-T** [comment]
Needs smoothing, very hard to do, but does need work, I'm assuming it's because the animation is conflicting as it's moving.

**Comment by aaronhowser1** [comment]
The selection area is also fairly small, maybe due to scaling on HD monitors? Mine is 1440p and those buttons are about a third of a fingernail-width tall (sorry, don't have a ruler :P)

**Comment by ThePaul-T** [comment]
About the same size for me tbh, I had noticed it but got caught up in other improvements. We will look at it but no promises for this release, depends how difficult it ends up being.

**Comment by Rushmead** [comment]
Yep this is all down to the size of the area that the buttons are in, unfortunately on this pack card size that is very very small. We give the title a set height on all the cards in order for the buttons not to be in different places for different packs, meaning that we have little height left to give to these buttons. 

A possible solution would be to move the version right under the name of the modpack. We could then remove some of the padding from  it which would then allow the buttons to take up as much height as possible from the bottom of the page.

That's probably the best solution at the moment, feel free to reply if you have another.
